<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWidget</class>
 <widget class="QWidget" name="SearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>180</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="filtersScrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>500</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="MultiStackedWidget" name="_searchFilters" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="ResultsViewWidget" name="_resultsView" native="true"/>
   </item>
  </layout>
  <action name="actionPreviousSearch">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../tagainijisho.qrc">
     <normaloff>:/images/icons/go-previous.png</normaloff>:/images/icons/go-previous.png</iconset>
   </property>
   <property name="text">
    <string>Previous search</string>
   </property>
  </action>
  <action name="actionNextSearch">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../tagainijisho.qrc">
     <normaloff>:/images/icons/go-next.png</normaloff>:/images/icons/go-next.png</iconset>
   </property>
   <property name="text">
    <string>Next search</string>
   </property>
  </action>
  <action name="actionResetSearch">
   <property name="icon">
    <iconset resource="../../tagainijisho.qrc">
     <normaloff>:/images/icons/eraser.png</normaloff>:/images/icons/eraser.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reset search...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MultiStackedWidget</class>
   <extends>QWidget</extends>
   <header>gui/MultiStackedWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ResultsViewWidget</class>
   <extends>QWidget</extends>
   <header>gui/ResultsViewWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../tagainijisho.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionPreviousSearch</sender>
   <signal>triggered()</signal>
   <receiver>SearchWidget</receiver>
   <slot>goPrev()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNextSearch</sender>
   <signal>triggered()</signal>
   <receiver>SearchWidget</receiver>
   <slot>goNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionResetSearch</sender>
   <signal>triggered()</signal>
   <receiver>SearchWidget</receiver>
   <slot>resetSearch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>goPrev()</slot>
  <slot>goNext()</slot>
  <slot>print()</slot>
  <slot>printPreview()</slot>
  <slot>printBooklet()</slot>
  <slot>printBookletPreview()</slot>
  <slot>tabExport()</slot>
  <slot>resetSearch()</slot>
 </slots>
</ui>
